service/src/main/resources/templates/index.html

```diff
@@ -1,106 +1,106 @@
 <!-- Copied from the swagger-ui static file -->
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
-  <head>
```

Comment:
are these changes purely formatting changes? (and waht did these changes?)

Reply:
oh weird. yeah i think they're just whitespace changes? i didn't do them on purpose. i can revert